### PR TITLE
GCP kubernetes playbooks

### PR DESCRIPTION
**Pull request by toddm92**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
New Playbooks:
* Prisma Cloud Remediation - GCP Kubernetes Engine Misconfiguration
* Prisma Cloud Remediation - GCP Kubernetes Engine Cluster Misconfiguration

They remediate the following Prisma Cloud policies:
* GCP Kubernetes Engine Clusters Basic Authentication is set to Enabled
* GCP Kubernetes Engine Clusters have HTTP load balancing disabled
* GCP Kubernetes Engine Clusters have Legacy Authorization enabled
* GCP Kubernetes Engine Clusters have Master authorized networks disabled
* GCP Kubernetes Engine Clusters have Network policy disabled
* GCP Kubernetes Engine Clusters have Stackdriver Logging disabled
* GCP Kubernetes Engine Clusters have Stackdriver Monitoring disabled
* GCP Kubernetes Engine Clusters have binary authorization disabled
* GCP Kubernetes Engine Clusters web UI/Dashboard is set to Enabled
* GCP Kubernetes cluster intra-node visibility disabled

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@toddm92 please fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

File: Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud.json [hunk 1]
```diff
@@ -40,7 +40,17 @@
 		"ee03a420-89d6-4745-a0ac-98878cb56cf4": "AWS EC2 Instance Misconfiguration",
 		"ef7c537b-72eb-42a7-bab7-cb2d22c76a0d": "AWS IAM Policy Misconfiguration",
 		"f53107a2-00b2-46fb-98a9-1f12262c7d44": "AWS IAM Policy Misconfiguration",
-		"fd4dae57-509e-4374-96d3-e136821fc3f3": "AWS IAM Policy Misconfiguration"
+		"fd4dae57-509e-4374-96d3-e136821fc3f3": "AWS IAM Policy Misconfiguration",
+                "6e125379-081e-4b06-a7ba-f04da2f0901a": "GCP Kubernetes Engine Misconfiguration",
```
Comment (mayagoldb) [inline Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud.json:44]: please fix indentation

File: Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json [hunk 2]
```diff
@@ -1995,6 +2005,494 @@
...
+						"accessor": "accountId",
+						"filters": [],
+						"root": "resource",
+						"transformers": []
+					},
+					"simple": ""
+				},
+				"Account Name": {
+					"complex": {
+						"accessor": "account",
```
Comment (mayagoldb) [inline Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json:2022]: Is this the right accessor for account name?

**Review by mayagoldb — CHANGES_REQUESTED** [review]

File: Packs/PrismaCloud/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "Prisma Cloud",
```
Comment (adi88d) [inline Packs/PrismaCloud/pack_metadata.json:2]: make sure the current pack metadata contains certification status, tags, price, usecases, etc
https://xsoar.pan.dev/docs/integrations/pack-docs#pack-description
https://xsoar.pan.dev/docs/integrations/packs-format#pack_metadatajson

File: Packs/PrismaCloud/IncidentTypes/incidenttype-GCP_Kubernetes_Engine_Misconfiguration.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "hoursR": 0,
+    "id": "GCP Kubernetes Engine Misconfiguration",
+    "locked": false,
+    "name": "GCP Kubernetes Engine Misconfiguration",
+    "playbookId": "Prisma Cloud Remediation - GCP Kubernetes Engine Misconfiguration",
+    "readonly": false,
+    "reputationCalc": 0,
+    "system": false,
+    "version": -1,
+    "fromVersion": "5.0.0",
```
Comment (adi88d) [inline Packs/PrismaCloud/IncidentTypes/incidenttype-GCP_Kubernetes_Engine_Misconfiguration.json:18]: fromversion value should be the same in all pack's entities (in classifier-Prisma_Cloud.json fromversion = 6.0.0 )

File: Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json [hunk 5]
```diff
@@ -0,0 +1,698 @@
+{
+	"TypeName": "",
+	"id": "GCP Kubernetes Engine Misconfiguration",
+	"kind": "details",
+	"fromVersion": "5.0.0",
```
Comment (adi88d) [inline Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json:5]: same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 6]
```diff
@@ -0,0 +1,1070 @@
+id: Prisma Cloud Remediation - GCP Kubernetes Engine Cluster Misconfiguration
+version: -1
+fromversion: 5.0.0
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:3]: same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml [hunk 7]
```diff
@@ -0,0 +1,669 @@
+id: Prisma Cloud Remediation - GCP Kubernetes Engine Misconfiguration
+version: -1
+fromversion: 5.0.0
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml:3]: same here

File: Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json [hunk 8]
```diff
@@ -0,0 +1,698 @@
...
+			"name": "Canvas",
+			"type": "canvas"
+		}
+	],
+	"typeId": "GCP Kubernetes Engine Misconfiguration",
+	"version": -1
+	},
+	"typeId": "GCP Kubernetes Engine Misconfiguration",
+	"version": -1,
+	"toVersion": "5.9.9"
```
Comment (adi88d) [inline Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json:697]: why you limit the version to 5.9.9 here?

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 9]
```diff
@@ -0,0 +1,1070 @@
...
+  "4":
+    id: "4"
+    taskid: 4b994469-67b1-4bcc-875b-a1b7922b49a4
+    type: regular
+    task:
+      id: 4b994469-67b1-4bcc-875b-a1b7922b49a4
+      version: -1
+      name: Get cluster details
+      description: Gets the details of a specific cluster.
+      script: GoogleKubernetesEngine|||gcloud-clusters-describe
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:83]:
```suggestion
      script: |||gcloud-clusters-describe
```
do not bind commands to specific instances

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 10]
```diff
@@ -0,0 +1,1070 @@
...
+    id: "17"
+    taskid: 245adf70-f841-4cfc-88da-185631bce956
+    type: regular
+    task:
+      id: 245adf70-f841-4cfc-88da-185631bce956
+      version: -1
+      name: Disable basic auth
+      description: Enable basic (username/password) auth for the cluster. Enable will
+        create user admin with generated password.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-muster-auth
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:567]:
```suggestion
      script: |||gcloud-clusters-set-muster-auth
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 11]
```diff
@@ -0,0 +1,1070 @@
...
+    id: "18"
+    taskid: 6e0531dd-e652-4380-8d51-cd2b9eaa5d94
+    type: regular
+    task:
+      id: 6e0531dd-e652-4380-8d51-cd2b9eaa5d94
+      version: -1
+      name: Disable legacy auth
+      description: Configuration for the legacy Attribute Based Access Control authorization
+        mode.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-legacy-auth
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:606]:
```suggestion
      script: |||gcloud-clusters-set-legacy-auth
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 12]
```diff
@@ -0,0 +1,1070 @@
...
+    taskid: 32ac86fd-c500-462b-809c-3abe9d1dc7eb
+    type: regular
+    task:
+      id: 32ac86fd-c500-462b-809c-3abe9d1dc7eb
+      version: -1
+      name: Enable networks auth
+      description: |-
+        Configuration options for the master authorized networks feature.
+        Enabled master authorized networks will disallow all external traffic to access Kubernetes master through HTTPS except traffic from the given CIDR blocks, Google Compute Engine Public IPs and Google Prod IPs.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-master-authorized-network
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:646]:
```suggestion
      script: |||gcloud-clusters-set-master-authorized-network
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 13]
```diff
@@ -0,0 +1,1070 @@
...
+    taskid: 732041a6-4c99-440f-8851-8138082365f1
+    type: regular
+    task:
+      id: 732041a6-4c99-440f-8851-8138082365f1
+      version: -1
+      name: Enable logging
+      description: Enable or Disable k8s stackdriver. Important - To use this functinality
+        the user should enable manually logging to "monitoring.googleapis.com/kubernetes"
+        manulally via the GCP console.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-k8s-stackdriver
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:727]:
```suggestion
      script: |||gcloud-clusters-set-k8s-stackdriver
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 14]
```diff
@@ -0,0 +1,1070 @@
...
+    taskid: 03b16ac7-cf90-4e63-8e39-6ee27115ab6a
+    type: regular
+    task:
+      id: 03b16ac7-cf90-4e63-8e39-6ee27115ab6a
+      version: -1
+      name: Enable monitoring
+      description: Enable or Disable k8s stackdriver. Important - To use this functinality
+        the user should enable manually logging to "monitoring.googleapis.com/kubernetes"
+        manulally via the GCP console.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-k8s-stackdriver
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:770]:
```suggestion
      script: |||gcloud-clusters-set-k8s-stackdriver
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 15]
```diff
@@ -0,0 +1,1070 @@
...
+  "23":
+    id: "23"
+    taskid: 93908a7a-ecdc-4ec9-8f95-024368b25523
+    type: regular
+    task:
+      id: 93908a7a-ecdc-4ec9-8f95-024368b25523
+      version: -1
+      name: Enable HTTP load balancing
+      description: Sets the addons for a specific cluster.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-addons
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:811]:
```suggestion
      script: |||gcloud-clusters-set-addons
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 16]
```diff
@@ -0,0 +1,1070 @@
...
+  "24":
+    id: "24"
+    taskid: 16e0934b-2c10-4118-8850-665a2e9b929b
+    type: regular
+    task:
+      id: 16e0934b-2c10-4118-8850-665a2e9b929b
+      version: -1
+      name: Disable UI/Dashboard
+      description: Sets the addons for a specific cluster.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-addons
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:851]:
```suggestion
      script: |||gcloud-clusters-set-addons
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 17]
```diff
@@ -0,0 +1,1070 @@
...
+  "25":
+    id: "25"
+    taskid: d67c07bd-d2f6-4d11-823d-9c232e30727d
+    type: regular
+    task:
+      id: d67c07bd-d2f6-4d11-823d-9c232e30727d
+      version: -1
+      name: Enable binary auth
+      description: Enable or Disable binary auth.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-binary-auth
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:891]:
```suggestion
      script: |||gcloud-clusters-set-binary-auth
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml [hunk 18]
```diff
@@ -0,0 +1,1070 @@
...
+  "26":
+    id: "26"
+    taskid: 8123d948-e03b-447b-8ec4-6bae03a73006
+    type: regular
+    task:
+      id: 8123d948-e03b-447b-8ec4-6bae03a73006
+      version: -1
+      name: Enable intra-node visibility
+      description: Enable or Disable for intra node visibility in cluster.
+      script: GoogleKubernetesEngine|||gcloud-clusters-set-intra-node-visibility
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:929]:
```suggestion
      script: |||gcloud-clusters-set-intra-node-visibility
```
same here

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml [hunk 19]
```diff
@@ -0,0 +1,669 @@
...
+    id: "12"
+    taskid: 1491bb10-b009-4dd8-8d1b-f12ff53a746a
+    type: regular
+    task:
+      id: 1491bb10-b009-4dd8-8d1b-f12ff53a746a
+      version: -1
+      name: Dismiss Prisma Cloud alert
+      description: Dismiss the alerts matching the given filter. Must provide either
+        policy IDs or alert IDs.
+      script: RedLock|||redlock-dismiss-alerts
```
Comment (adi88d) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml:462]:
```suggestion
      script: |||redlock-dismiss-alerts
```
same here

**Review by adi88d — CHANGES_REQUESTED** [review]

File: Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json (hunk 2 above)
Comment (toddm92) [reply Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json:2022]: it's the same for "GCP Compute Engine Misconfiguration"

File: Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud.json (hunk 1 above)
Comment (toddm92) [reply Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud.json]: fixed.

File: Packs/PrismaCloud/IncidentTypes/incidenttype-GCP_Kubernetes_Engine_Misconfiguration.json (hunk 4 above)
Comment (toddm92) [reply Packs/PrismaCloud/IncidentTypes/incidenttype-GCP_Kubernetes_Engine_Misconfiguration.json]: fixed.

File: Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json (hunk 5 above)
Comment (toddm92) [reply Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json]: fixed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 6 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: fixed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml (hunk 7 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml]: fixed.

File: Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json (hunk 8 above)
Comment (toddm92) [reply Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json:697]: I didn't change the value of `toVersion`.  The value `5.9.9` is from the Demisto export.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 9 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml:83]: this is an integration cmd and again a result from the Demisto export.  do you still want the change?

File: Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json (hunk 8 above)
Comment (toddm92) [reply Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json:697]: is this to support versions prior to 6.0?

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 9 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 10 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 11 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 12 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 13 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 14 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 15 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 16 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 17 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml (hunk 18 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml (hunk 19 above)
Comment (toddm92) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Misconfig.yml]: changed.

File: Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json (hunk 2 above)
Comment (mayagoldb) [reply Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json:2022]: Should be "accountname", please change in all relevant files - layouts, classification& mapping etc.

Comment (toddm92) [reply Packs/PrismaCloud/Classifiers/classifier-Prisma_Cloud_5_9_9.json]: changed.

File: Packs/PrismaCloud/pack_metadata.json (hunk 3 above)
Comment (toddm92) [reply Packs/PrismaCloud/pack_metadata.json:2]: updated.

**Comment by adi88d** [comment]
@toddm92 sorry for asking you to changed the `fromVersion` values it was not necessary, i will revert your changes

**Comment by mayagoldb** [comment]
@yaron-libman please review playbooks and release notes

**Review by adi88d — APPROVED** [review]

File: Packs/PrismaCloud/Layouts/layoutscontainer-GCP_Kubernetes_Engine_Misconfiguration.json [hunk 20]
```diff
@@ -0,0 +1,704 @@
...
+    "itemVersion": "1.2.3",
+    "locked": false,
+    "mobile": null,
+    "name": "GCP Kubernetes Engine Misconfiguration",
+    "packID": "",
+    "propagationLabels": [],
+    "quickView": null,
+    "sortValues": null,
+    "system": false,
+    "toServerVersion": "",
```
Comment (yaakovi) [inline Packs/PrismaCloud/Layouts/layoutscontainer-GCP_Kubernetes_Engine_Misconfiguration.json:702]:
```suggestion
    "toServerVersion": "",
    "fromVersion": "6.0.0",
```

**Review by mayagoldb — APPROVED** [review]

**Comment by kirbles19** [comment]
@yaakovi Done.

**Pull request merged**